bgp/modules/terms.py

```diff
@@ -6,16 +6,33 @@
 from lxml import etree
 
 
-STOP_WORDS = set("""a about above after again against all almost also am among an and
-    any are around as at be became because been before being below between both but by
-    can could did do does doing don down during each ever far few for from further had
-    has have having he her here hers herself him himself his how i if in into is it its
-    itself just like many may me more most much my myself new no nor not now of off
-    often on once one only or other our ours ourselves out over own s same see seems
-    she should so some such t than that the their theirs them themselves then there
-    these they things this those three through to too two under until up very was we
-    well were what when where which while who whom why will with would you your yours
-    yourself yourselves""".split())
+STOP_WORDS = {
```

Comment:
```suggestion
STOP_WORDS = set("""'d 'll 'm 're 's 've a about above across after afterwards
again against all almost alone along already also although always am among
amongst amount an and another any anyhow anyone anything anyway anywhere are
around as at back be became because become becomes becoming been before
beforehand behind being below beside besides between beyond both bottom but by
ca call can cannot could did do does doing done down due during each eight
either eleven else elsewhere empty enough even ever every everyone everything
everywhere except few fifteen fifty first five for former formerly forty four
from front full further get give go had has have he hence her here hereafter
hereby herein hereupon hers herself him himself his how however hundred i if
in indeed into is it its itself just keep last latter latterly least less made
make many may me meanwhile might mine more moreover most mostly move much must
my myself n't name namely neither never nevertheless next nine no nobody none
noone nor not nothing now nowhere n‘t n’t of off often on once one only onto
or other others otherwise our ours ourselves out over own part per perhaps
please put quite rather re really regarding same say see seem seemed seeming
seems serious several she should show side since six sixty so some somehow
someone something sometime sometimes somewhere still such take ten than that
the their them themselves then thence there thereafter thereby therefore
therein thereupon these they third this those though three through throughout
thru thus to together too top toward towards twelve twenty two under unless
until up upon us used using various very via was we well were what whatever
when whence whenever where whereafter whereas whereby wherein whereupon
wherever whether which while whither who whoever whole whom whose why will
with within without would yet you your yours yourself yourselves ‘d ‘ll ‘m ‘re
‘s ‘ve ’d ’ll ’m ’re ’s ’ve""".split())
```